processor/signoztailsampler/testdata/tail_sampling_config.yaml

```diff
@@ -1,107 +1,30 @@
-tail_sampling:
+signoz_tail_sampling:
   decision_wait: 10s
   num_traces: 100
   expected_new_traces_per_sec: 10
   policies:
     [
         {
           name: test-policy-1,
-          type: always_sample
+          type: policy_group,
+          priority: 1,
+          policy_filter: {
+            string_attributes: {
+              key: source,
+              values: ["security"]
+            }
+          }, 
+          sampling: {sampling_percentage: 100},
+          sub_policies: [{
+            name: condition-1, 
+            type: policy_group, 
```

Comment:
Can you add additional config tests with different policy types? I think we only have three or four.

Reply:
See my comment here https://github.com/SigNoz/signoz-otel-collector/pull/111/files#r1146944199. In the config.go, there are five policy types defined, and any config is valid as long it belongs to one of them. If you are saying that there should only be `policy_group`, you should remove the rest.